templates/zerver/help/pin-a-stream.md

```diff
@@ -0,0 +1,45 @@
+# Pinning a stream
+
+Pinning a stream will put that stream at the top of your streams sidebar. If you are subscribed to a lot of streams, pinning your favorite streams can be a good way to keep things organized.
+
+## How to pin a stream
+
+There are two easy ways to pin a stream:
+
+### Pinning a stream using the sidebar
+
+The quickest way to pin a stream is by using the **Pin stream ... to top** feature on the sidebar.
+
+1) On the sidebar, hover over the stream you want to pin and press the arrow to the right of the stream name.
+
+![Arrow](/static/images/help/arrow.png)
+
+2) A menu will pop up. Press **Pin stream ... to top**.
+
+![Pin1](/static/images/help/pin1.png)
+
+3) The stream you just pinned will now be at the top of your streams list.
+
+![Pin2](/static/images/help/pin2.png)
+
+### Pinning a stream using the stream settings:
+
+The other way to pin a stream is by ticking the **Pin stream to top of left sidebar** box in the stream settings.
+
+1) Open your subscription settings by pressing the gear at the top of the left sidebar.
+
+![Browse streams](/static/images/help/streams-1.png)
+
+2) Now select the stream you want to pin and tick the box **Pin stream to top of left sidebar**.
+
+![Pin3](/static/images/help/pin3.png)
+
+3) The stream you just pinned will now be at the top of your streams list.
+
+![Pin2](/static/images/help/pin2.png)
```

Comment:
Instead of showing the same image 2 times I'd rather put it as a result of any of those methods once.